src/pubtools/_marketplacesvm/tasks/delete/command.py

```diff
@@ -173,9 +172,10 @@ def _delete(
                     self.update_rhsm_metadata(push_item)
                     return pi
                 except Exception as exc:
-                    # If we failed the image might not exist, not necessarily an error
+                    # If we failed, the image might not exist, not necessarily an error
                     delete_error = exc
                     failed_marketplace.append(marketplace)
+                    pass
```

Comment:
Instead of pass for generic Exception, could we catch some more specific exception?
Just not sure what t boto reports here if you try to remove non-existing image.

But idempotent implementation should not fail to remove non-existing images. :-)

Reply:
+1 for specific Exception catch

Reply:
Updated with changes to this.